python/cac_tripplanner/templates/admin/cac-geocoding-map.js

```diff
@@ -0,0 +1,22 @@
+{% extends 'gis/admin/osm.js' %}
+
+{% block extra_layers %}
+
+var $address = $('[name=address]');
+var module = {{ module }};
+var map = module.map;
+var projLatLng = new OpenLayers.Projection('EPSG:4326');
+var projWM = new OpenLayers.Projection('EPSG:900913'); // 3857 doesn't work here for some reason
```

Comment:
Cool, nice find! I'll update the comment with the reason, but will keep the code as-is, since finding the right place in the template to add the override will definitely be more trouble than it's worth.